programs/address-lookup-table/src/instruction.rs

```diff
@@ -84,6 +84,7 @@ pub fn create_lookup_table(
     authority_address: Pubkey,
     payer_address: Pubkey,
     recent_slot: Slot,
+    authority_is_signer: bool,
```

Comment:
Do you want to remove this argument once the feature is enabled on all clusters? If so, it'll be easier for clients on the transition if there are two functions, one that declares the authority as a signer, and one that doesn't. You could have the existing `create_lookup_table_signed` require the signature, mark it as deprecated, and also have `create_lookup_table` which doesn't.

Reply:
Good call, added